src/_P005_DHT.ino

```diff
@@ -151,7 +151,7 @@ bool P005_do_plugin_read(struct EventStruct *event) {
   digitalWrite(Plugin_005_DHT_Pin, LOW);              // Pull low
   
   switch (Par3) {
-    case P005_DHT11:  delay(18); break;  // minimum 18ms
+    case P005_DHT11:  delay(19); break;  // minimum 18ms
```

Comment:
I always get a bit nervous by these kind of fixes.
Delay from 18 to 19 msec....
If this really is needed, then the timing is way too critical.

Reply:
we discussed this once, it should be 18 minimum, so 19 is not big deal, for dht22 minimum is 1ms and 2ms works ok, 
actually it is hard to buy dht11

Reply:
> we can add delay(0) before turning of interrupts, what do you think?

That sounds like a good idea.

Reply:
@TD-er about 1ms more, it is also better for longer wires

Reply:
@TD-er bad idea with delay, because we have already delays just before

Reply:
Well, it depends, is the delay from the switch statement just critical as minimum, or also for the maximum?

Reply:
@TD-er first delay is start condition - request for data and it is minimum critical, every next should not be more than in documentation so critical for maximum.
we still don't know how accurate delays are in esp8266 from poor quality boards to nice ones.